4-perftools/Dockerfile.rhel7

```diff
@@ -38,7 +37,7 @@ USER 0
 
 WORKDIR ${HOME}
 
-COPY contrib/etc/scl_enable /root/etc/scl_enable
+COPY root/root/etc/scl_enable /root/etc/scl_enable
```

Comment:
> USER 0 is default, so this command is redundant too and might not affect the COPY.

I was thinking maybe it has an effect on some obscure container platform, otherwise the instruction doesn't make sense. But I agree it's most likely just an oversight. I'll add a separate commit to get rid of the superfluous copies.